packages/devtools_app/lib/src/service/vm_service_wrapper.dart

```diff
@@ -1317,18 +1314,18 @@ extension VmServicePrivate on VmServiceWrapper {
   }
 
   /// Forces the VM to perform a full garbage collection.
-  Future<Success> collectAllGarbage() => _privateRpcInvoke(
+  Future<Success?> collectAllGarbage() => _privateRpcInvoke(
```

Comment:
why does this have to be nullable if `_privateRpcInvoke` has a non-nullable return type? Same question for RPCs below

Reply:
PortList.parse returns PortList?
So, T=PortList?, i.e. return type is nullable

Reply:
If that is the case, then the `parser` parameter in `_privateRpcInvoke` should be: `required T? Function(Map<String, dynamic>?) parser,` (note the T? instead of T). And `_privateRpcInvoke` should also return a nullable type: `Future<T?>`

Reply:
done